Gathered fixes from abandoned PRs. (#6762)
magento/devdocs: src/guides/v2.3/javascript-dev-guide/javascript/js_init.md

## Calling JS components requiring initialization in JS files {#js_widget_init}

> To call a widget in JS code, use a notation similar to the following ([accordion]({{ page.baseurl }}/javascript-dev-guide/widgets/widget_accordion.html) widget is initialized on the `[data-role=example]` element as illustration):

Reviewer: This doesn't even read correctly. Needs work.

> To call a widget in JS code, use a notation similar to the following ([accordion]({{ page.baseurl }}/javascript-dev-guide/widgets/widget_accordion.html) widget is initialized on the `[data-role=example]` element as illustration):

Reviewer 2: ```suggestion
To call a widget in JS code, use a notation similar to the following ([accordion]({{ page.baseurl }}/javascript-dev-guide/widgets/widget_accordion.html) widget that is initialized on the `[data-role=example]` element below:
```

Author: Fixed
